sonar-java-plugin/src/test/java/org/sonar/plugins/java/JavaSensorTest.java

```diff
@@ -277,6 +277,15 @@ void does_not_insert_SymbolicExecutionVisitor() throws IOException {
         "ParameterReassignedToCheck"
       );
   }
+  
+  @Test
+  void info_log_when_no_SE_rules_enabled() throws IOException {
+    Loggers.get(PerformanceMeasure.class).setLevel(LoggerLevel.INFO);
+    assertJasperIsInvoked(new MapSettings());
+    String debugLogs = String.join("\n", logTester.logs(LoggerLevel.INFO));
+    assertThat(debugLogs).contains("No rules with 'symbolic-execution' tag were enabled,"
+      + " the Symbolic Execution Engine will not run during the analysis.");
```

Comment:
No need to get and join all log lines. You can simply do `assertThat(logTester.logs(LoggerLevel.INFO)).contains("My log message");`